src/resolver/index.spec.js

```diff
@@ -0,0 +1,9 @@
+import resolver from '.';
+
+describe('With options', () => {
```

Comment:
please also test without options (it should work as it did before, and it won't with your code, showing that you should fix it).